panel/src/helpers/string.js

```diff
@@ -80,7 +80,7 @@ export function stripHTML(string) {
 export function template(string, values = {}) {
   const resolve = function (parts, data = {}) {
     const part  = escapeHTML(parts.shift());
-    const value = data[part] || null;
+    const value = data[part] ?? null;
```

Comment:
Wait, since when is ?? supported in JS?

Reply:
Yep! While working in JSFiddle, it was not working in our panel, but it seems to be working now. I guess it happened by updating our dependencies.

https://caniuse.com/mdn-javascript_operators_nullish_coalescing

Reply:
Since ES2020. 🙂 See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

There's also optional chaining, which might peak your interest as well!
```js
const kirby = {
  creator: {
    name: 'Bastian'
  }
}

console.log(kirby.creator?.nonExistingKey)
// expected output: undefined

console.log(kirby.someNonExistentMethod?.())
// expected output: undefined

``